src/azure-cli/azure/cli/command_modules/acr/replication.py

```diff
@@ -35,11 +39,37 @@ def acr_replication_create(cmd,
         raise CLIError('Replication could not be created in the same location as the registry.')
 
     from msrest.exceptions import ValidationError
+    ReplicationType = cmd.get_models('Replication')
+
+    # version below adds region_endpoint_enabled and a breaking change where some create params are grouped into a model
+    grouped_params_create_api_version = "2019-12-01-preview"
```

Comment:
What's preventing us from updating the api-version for all commands?

Reply:
So for some reason, their CI might use a different profile which uses an older api version. Versions older than this version have a different signature for create.

This is why I check for the api version.